src/components/v2/common/ReactSelect.utils.tsx

```diff
@@ -4,6 +4,21 @@ import { ReactComponent as Check } from '../assets/icons/ic-check.svg';
 import { components } from 'react-select';
 import Tippy from '@tippyjs/react';
 
+export const getCustomOptionSelectionStyle = (styleOverrides = {}) => {
+    return (base, state) => ({
+        ...base,
+        backgroundColor: state.isSelected ? '#f0f7ff' : state.isFocused ? 'var(--N100)' : 'white',
```

Comment:
use respective `var` instead of color code directly.

Reply:
Done